server/server.js

```diff
@@ -45,13 +46,14 @@ var uploadS3 = multer({
       getFieldname(null, {fieldName: file.fieldname});
     },
     key: (req, file, createAWSName) => {
-      createAWSName(null, Date.now().toString() + '-' + file.originalname);
+      console.log('success on multer key:', file.originalname);
+      createAWSName(null, 'test-file');
```

Comment:
DOesn't matter now, but were you going to add in the timestamped file name again?